keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -177,22 +177,27 @@ type PersistentVolumeClaimStatus struct {
   ...
   // The VolumeAttributesClassName string cannot be empty
   VolumeAttributesClassName string
-  ModifyVolumeStatus *PersistentVolumeClaimModifyVolumeStatus
+  ModifyVolumeStatus ModifyVolumeStatus
   ...
 }
 
+type ModifyVolumeStatus struct {
+  ActualClassName string
+  TargetClassName string
+  Status *PersistentVolumeClaimModifyVolumeStatus
+}
+
 // +enum
 type PersistentVolumeClaimModifyVolumeStatus string
 
 const (
   // When modify volume is complete, the empty string is set by modify volume controller.
-  PersistentVolumeClaimNoModifyVolumeInProgress PersistentVolumeClaimModifyVolumeStatus = ""
+  PersistentVolumeClaimNoModifyVolumeInSucceeded PersistentVolumeClaimModifyVolumeStatus = ""
   // State set when modify volume controller starts modifying the volume in control-plane
+  // Non terminal errors should also keep this status without modification
   PersistentVolumeClaimControllerModifyVolumeInProgress PersistentVolumeClaimModifyVolumeStatus = "ControllerModifyVolumeInProgress"
   // State set when modify volume has failed in modify volume controller with a terminal error.
```

Comment:
A better description may be "when the request has been rejected as invalid by the CSI driver. To resolve this error, the PersistentVolumeClaim needs to specify a valid VolumeAttributesClass."